src/main/resources/templates/admin.html

```diff
@@ -156,7 +156,7 @@ <h2 class="card-title mt-md-1 mt-0 mb-1 d-inline-block">Manage Admins</h2>
                     <div class="row m-auto pt-3 pb-3 d-flex">
                         <div class="col-lg-9 col-md-8 col-12 p-0 text-center text-md-left">
                             <h2 class="mt-md-1 mt-0 d-inline-block">Manage Person</h2>
-                            <br class="d-md-none"><h3 class="d-inline-block ml-0 ml-lg-1 mt-n1 mt-lg-0 align-middle">{{currentManagePerson}}</h3>
+                            <br class="d-md-none"><div class="h3 d-inline-block ml-0 ml-lg-1 mt-n1 mt-lg-0 align-middle">{{currentManagePerson}}</div>
```

Comment:
Not too sure I like this hack of having a h3 class to avoid the ADA issue. I think we can use an ng-if attribute to fix the ADA issue because if there is no "currentManagePerson" to display, then it will remove the HTML element from the DOM.